styles/components/_menu.scss

```diff
@@ -1,60 +1,66 @@
 .page__container {
-  width: 100vw;
-  height: calc(100vh - 2rem);
-  background-color: white;
-  text-align: center;
+  display: flex;
+  height: 100vh;
+  background: url(/static/images/ui/background.png) no-repeat;
+  background-size: cover;
 }
 
 .menu-window {
   display: flex;
   flex-direction: column;
   justify-content: space-between;
+  margin: 80px 0 80px 80px;
   &__header {
-    position: relative;
-    display: flex;
-    flex-direction: column;
-    padding: 32px;
-  }
-  &__heading {
-  }
-  &__description {
+    font-family: Roboto;
+    font-weight: 700;
   }
 }
 
 .menu {
-  position: relative;
   display: flex;
-  align-items: center;
-  justify-content: space-between;
-  padding: 32px;
-  &__graphic {
-    &-container {
-      width: 30vw;
-      height: 50vh;
-      background-color: cadetblue;
-    }
-  }
-  &__buttons {
-    display: flex;
-    flex-direction: column;
-  }
+  flex-direction: column;
+  flex-grow: 1;
+  text-align: start;
   &__button {
-    &--rules {
-      position: absolute;
-      right: 2rem;
-      bottom: 0;
-      width: 2rem;
-      height: 2rem;
+    border: none;
+    margin-top: 50px;
+    background: none;
+    font-family: Roboto;
+    font-size: 24px;
+    font-weight: 700;
+    outline: none;
+    text-align: start;
+    &:hover {
+      color: #f00;
+      cursor: pointer;
     }
   }
 }
+.start-button {
+  width: 25%;
+  height: 13%;
+  align-self: flex-end;
```

Comment:
when it is a part of the menu window container, you can give him align-self:center, so it gets positiones automatically in the middle of the screen, so we dont need to rely on margins